packages/web/src/components/gcds-radios/stories/gcds-radios.stories.tsx

```diff
@@ -0,0 +1,312 @@
+  ${args.required ? `required` : null}
+  ${args.disabled ? `disabled` : null}
+  ${args.value ? `value="${args.value}"` : null}
+  ${args.validateOn != 'blur' ? `validate-on="${args.validateOn}"` : null}
+  ${args.lang != 'en' ? `lang="${args.lang}"` : null}
+>
+</GcdsRadios>
+`.replace(/\s\snull\n/g, '');
+
+const TemplatePlayground = args =>
+  `
+<!-- Web component code (Angular, Vue) -->
+<gcds-radio-group
+  name="${args.name}"
+  options='${args.options}'
+>
+</gcds-radio-group>
+`.replace(/\s\snull\n/g, '');
+
+export const Default = Template.bind({});
+Default.args = {
+  name: 'radioDefault',
+  legend: 'Legend',
+  options: `[
+    { "label": "Label for radio 1", "id": "radio1", "value": "radio1"},
+    { "label": "Label for radio 2", "id": "radio2", "value": "radio2"}
+  ]`,
+  hint: '',
+  errorMessage: '',
+  required: false,
+  disabled: false,
+  value: '',
+  validateOn: 'blur',
+  lang: 'en',
+};
+
+export const Hint = Template.bind({});
+Hint.args = {
+  name: 'radio',
+  legend: 'Legend',
+  options: `[
+    { "label": "Label for radio 1", "id": "radio1", "value": "radio1", "hint": "Description or example to make the option clearer."},
+    { "label": "Label for radio 2", "id": "radio2", "value": "radio2", "hint": "Description or example to make the option clearer."}
+  ]`,
+  hint: 'Hint text',
+  errorMessage: '',
+  required: false,
+  disabled: false,
+  value: '',
+  validateOn: 'blur',
+  lang: 'en',
+};
+
+export const Required = Template.bind({});
+Required.args = {
+  name: 'radio',
+  legend: 'Legend',
+  options: `[
+    { "label": "Label for radio 1", "id": "radio1", "value": "radio1"},
+    { "label": "Label for radio 2", "id": "radio2", "value": "radio2"}
+  ]`,
+  hint: '',
+  errorMessage: '',
+  required: true,
+  disabled: false,
+  value: '',
+  validateOn: 'blur',
+  lang: 'en',
+};
+
+export const Disabled = Template.bind({});
+Disabled.args = {
+  name: 'radio',
+  legend: 'Legend',
+  options: `[
+    { "label": "Label for radio 1", "id": "radio1", "value": "radio1"},
+    { "label": "Label for radio 2", "id": "radio2", "value": "radio2"}
+  ]`,
+  hint: '',
+  errorMessage: '',
+  required: false,
+  disabled: true,
+  value: '',
+  validateOn: 'blur',
+  lang: 'en',
+};
+
+export const CheckedProp = Template.bind({});
+CheckedProp.args = {
+  name: 'radio',
+  legend: 'Legend',
+  options: `[
+    { "label": "Label for radio 1", "id": "radio1", "value": "radio1"},
+    { "label": "Label for radio 2", "id": "radio2", "value": "radio2", "checked": true}
+  ]`,
+  hint: '',
+  errorMessage: '',
+  required: false,
+  disabled: false,
+  value: '',
+  validateOn: 'blur',
+  lang: 'en',
+};
+
+export const CheckedValue = Template.bind({});
+CheckedValue.args = {
+  name: 'radio',
+  legend: 'Legend',
+  options: `[
+    { "label": "Label for radio 1", "id": "radio1", "value": "radio1"},
+    { "label": "Label for radio 2", "id": "radio2", "value": "radio2"}
+  ]`,
```

Comment:
```suggestion
  options: `[
    { "label": "Label for radio 1", "id": "radio1", "value": "radio1"},
    { "label": "Label for radio 2", "id": "radio2", "value": "radio2"}
  ]`,
```

This is a random choice but we can also test out if giving an object here (instead of a string) will render the component